org.eclipse.cpacep/src/org/eclipse/cpacep/util/CPACEPConnector.java

```diff
@@ -71,7 +71,7 @@ private void initializeBaseCommandLine() {
 		baseCli.add(" -stats");
 	}
 
-	private String executeCommand(String command) {
+	public static String executeCommand(String command) {

```

Comment:
Try to avoid static methods, because they make the code a lot less flexible.
Instead, you should create a `CPACEPConnector` instance for your test. You could add a custom constructor for that.